var/spack/repos/builtin/packages/acl/package.py

```diff
@@ -23,14 +27,33 @@ class Acl(AutotoolsPackage):
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
-    depends_on("attr")
+    depends_on("bash", type="build")
     depends_on("gettext")
 
+    depends_on("attr")
+    # The configure step fails to find attr/xattr.h for these versions of acl unless this exact
+    # version of attr is used.
+    depends_on("attr@2.4.47", when="@:2.2.51")
+
+    # In 2.2.53 and later, the codebase provides a nice autogen script.
+    @when("@2.2.53:")
+    def autoreconf(self, spec, prefix):
+        which("bash")("./autogen.sh")
```

Comment:
```suggestion
    def autoreconf(self, spec, prefix):
        autogen = Executable("./autogen.sh")
        autogen()
```
Acl's `autogen.sh` doesn't make use of any bash-isms so we can launch this instead with a standard `sh` script which is more compatible for folks using distributions like Alpine.